docs/components/concepts/encryption-at-rest.md

```diff
@@ -0,0 +1,74 @@
+---
+id: encryption-at-rest
+title: "Encryption at rest"
+description: "Camunda 8 SaaS cluster data at rest is protected using Google Cloud Platform (GCP) encryption with a provider-managed encryption key."
+keywords:
+  [
+    "encryption",
+    "encryption at rest",
+    "encryption key",
+    "provider key",
+    "software key",
+    "hardware key",
+  ]
+---
+
+<span class="badge badge--cloud">Camunda 8 SaaS only</span>
+
+Camunda 8 SaaS cluster data is encrypted at rest to provide security and protection for your data.
+
+## Overview
+
+By default, Camunda 8 SaaS cluster data at rest is protected with a provider-managed encryption key using [Google Cloud Platform (GCP) encryption](https://cloud.google.com/docs/security/encryption/default-encryption). The encryption key is owned and managed by GCP.
+
+Enterprise customers requiring a higher level of protection can select a dedicated Camunda-managed software or hardware (HSM) encryption key when creating a new cluster. The encryption key is managed by Camunda using Google Cloud Key Management Service (KMS).
+
+- You can only select the encryption type when [creating a cluster](/docs/components/console/manage-clusters/create-cluster.md). You cannot change the encryption type after cluster creation.
+- You can configure encryption keys on a per-cluster basis so that each cluster has a dedicated encryption key. Encryption keys can be configured for all cluster versions.
+- You can view cluster encryption key details in **Cluster details** on the **Console Overview** tab.
+
+:::note
+You cannot use Camunda-managed software or hardware encryption keys for encryption of backups. Backups use the default provider GCP encryption.
```

Comment:
```suggestion
Backups use the default provider GCP encryption.
```

This is not something the customer can influence. So we can keep the note concise that the Backups use the other encryption mechanisms.

@felix-mueller would be probably good to at this on our roadmap to encrypt backups too. This will certainly be needed by customer requirements in the end. We already have document handling topic coming up where we could tackle this potentially.